### DISCUSSION TITLE
Point-to-multipoint communication with ipsec

### DISCUSSION THREAD

**Discussion by Murik1990** [strongswan/strongswan#1858] [Q&A]
Hello Everybody. I want to control several remote points from one point. All remote points were connected to each other through one point. Can I do this with IPsec?

**Comment by tobiasbrunner** [comment]
> I want to control several remote points from one point.

Control how?

> All remote points were connected to each other through one point.

That sounds like a classic [hub-and-spoke topology](https://wiki.strongswan.org/projects/strongswan/wiki/SubnetsBehindMoreThanTwoGateways). Or even a roadwarrior setup if the clients don't have a subnet behind them.
